buildSrc/subprojects/jvm/src/main/kotlin/gradlebuild/propagated-env-variables.kt

```diff
@@ -41,6 +41,7 @@ val propagatedEnvironmentVariables = listOf(
 
     // Used by Gradle test infrastructure
     "REPO_MIRROR_URL",
+    "SDKMAN_CANDIDATES_DIR",
```

Comment:
Can you add a more concrete comment that this is used for finding Java installations for tests?